modules/serverless-functions-secrets-configmaps-interactively.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
+
+[id="serverless-functions-secrets-configmaps-interactively_{context}"]
+= Modifying function access to secrets and config maps interactively
+
+You can manage the secrets and config maps accessed by your function by using the `kn func config` interactive utility.
+
+.Procedure
+
+. Run the following command in the function project directory:
++
+[source,terminal]
+----
+$ kn func config
+----
++
+Alternatively, you can specify the function project directory using the `--path` or `-p` option.
```

Comment:
Do you need the brackets like in the _with specialized commands_ topic? 

`kn func config [-p <path>]`

Reply:
The brackets are there per the convention for presenting a command, same as in, say, the Unix man pages. The brackets shouldn't be entered when actually using the option.

Reply:
OK. I see other places in the docs that use brackets like that. I will leave the decision to you and your team whether to include them. Personally, I would have copied the brackets into the command I am executing. But, I am not the target audience for these docs. 

I would have entered: `kn func config [-p my-function-project]`. Would this work?